chain/indexer/src/streamer/utils.rs

```diff
@@ -43,6 +43,7 @@ pub(crate) async fn convert_transactions_sir_into_local_receipts(
                     },
                     prev_block_gas_price,
                     true,
+                    block.header.latest_protocol_version,
```

Comment:
AFAIU this is actually the latest protocol version supported by the block producer. This is *not* the protocol version approved by consensus, that the current epoch is running with. Source:
https://github.com/near/nearcore/blob/cbf977aef7685c705df6fe1c7bce142eadf7076a/core/primitives/src/block_header.rs#L69-L70

The epoch protocol version should I think be obtained with something like this code:
https://github.com/near/nearcore/blob/cbf977aef7685c705df6fe1c7bce142eadf7076a/chain/chain/src/chain.rs#L879-L880

Reply:
Good catch!

Reply:
Hmm makes sense. This being said, if the good fix is hard to do, we should add a big comment saying that this is intentionally wrong, to avoid other people copy-pasting that into protocol-breaking code later :)
(If we already had an instance of this in our code then I'd have assumed that I was misunderstanding the meaning of latest_protocol_version when reviewing)

Reply:
+1, I would have the same impression :) so the way to get proto versions should be consistent across the code (unless the params to call function is not available and there is a big comment about it)